### PR TITLE
Hide predeployed accounts details

### PR DESCRIPTION
**Pull request by LucasLvy**
## Usage related changes

- Users can now hide the details of the predeployed accounts. Default behavior is unchanged

## Development related changes

- Devnet output can be less verbose

## Checklist:

- [x] Applied formatting - `./scripts/format.sh`
- [x] No linter errors - `./scripts/lint.sh`
- [x] Performed code self-review
- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Documented the changes
- [x] Linked the issues which this PR resolves
- [x] Updated the tests
- [x] All tests are passing

### CODE REVIEW

File: starknet_devnet/starknet_wrapper.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ def __init__(self, config: DevnetConfig):
         self.starknet: Starknet = None
         self.__current_cached_state = None
         self.__initialized = False
+        self.hide_predeployed_accounts = config.hide_predeployed_accounts
```
Comment (FabijanC) [inline starknet_devnet/starknet_wrapper.py:86]: I would say this is not necessary because `self.config = config` is already done.

**Review by FabijanC — COMMENTED** [review]
Something like this is proposed here: https://github.com/Shard-Labs/starknet-devnet/issues/147

**Comment by LucasLvy** [comment]
> Something like this is proposed here: #147

Anyone working on it ?

**Comment by FabijanC** [comment]
> > Something like this is proposed here: #147
> 
> Anyone working on it ?

No

**Review by mikiw — APPROVED** [review]
Looks fine for me.

**Pull request merged**